2.4 i18n - post merge string fixes

esbrandt opened the pull request:
Minor fixes for errors found while translating strings, post merge https://github.com/mixxxdj/mixxx/pull/3728

esbrandt committed:
Add missing line break in multi-line user facing string

esbrandt committed:
Remove unneeded html formatting for user facing string, and make it a bit less technical

Holzhaus commented on src/encoder/encoderfdkaac.cpp:
```diff
@@ -261,10 +261,10 @@ int EncoderFdkAac::initEncoder(int samplerate, QString* pUserErrorMessage) {
             // and include link in message below
             *pUserErrorMessage = QObject::tr(
-                    "<html>Mixxx cannot record or stream in AAC "
+                    "Mixxx cannot record or stream in AAC "
```
Are you sure this isn't to force rich formatting?

uklotzde replied:
This also applies to 2.3. If we fix it we should fix it there first.

uklotzde commented:
The first commit belongs into main and should be merged asap. The second commit should be reverted and published as a separate PR.

esbrandt committed:
Revert "Remove unneeded html formatting for user facing string, and make it a bit less technical"

This reverts commit 72785c1990c2ba6ce4c4f947ee002fdf5143e3a3.

esbrandt committed:
Merge remote-tracking branch 'upstream/main' into 2.4-i18n

esbrandt committed:
Update translation template
Found 2718 source text(s) (2 new and 2716 already existing)
Removed 4 obsolete entries

esbrandt commented:
> The first commit belongs into main and should be merged asap. The second commit should be reverted and published as a separate PR.

Done

uklotzde reviewed (approved):
LGTM (waiting for CI)

uklotzde commented:
No need to wait for the build-checks.
